### ISSUE TITLE
Error parsing data-labels in `createStatus` action

### ISSUE THREAD

**Issue by dahaupt** [in2code-de/femanager#509]
The action `\In2code\Femanager\Controller\NewController::createStatusAction` is not assigning the variable `jsLabels` which results in the following error after a successful registration:
```
SyntaxError
JSON.parse: unexpected end of data at line 1 column 1 of the JSON data
```
I propose to remove the layout from the template if that has no side effects, which is the case in our project. Alternatively, the attribute `data-labels` could be set only, if `jsLabels` is set.

**Comment by BastiLu** [comment]
Solved.

**Comment by abvdveen** [comment]
This is not solved in 7.2.2. The function createStatusAction still hasn't got $this->assignForAll(), but Layouts/Default.html still has got data-labels="{jsLabels}".
So, after registration, in the console you get 'Uncaught SyntaxError: Unexpected end of JSON input'

**Comment by sbusemann** [comment]
this was published with 7.2.3
